_docs/reference/libs-non-rel-raw.csv

```diff
@@ -12,7 +12,51 @@ charmed-kubeflow-chisme,team,https://pypi.org/project/charmed-kubeflow-chisme/,,
+charms.templating.jinja2,legacy,https://pypi.org/project/charms.templating.jinja2/,https://pythonhosted.org/charms.templating.jinja2/,https://github.com/juju-solutions/charms.templating.jinja2,PyPI,,,Legacy library for Jinja templating in reactive charms.
+coordinated-workers,team,https://pypi.org/project/coordinated-workers/,,https://github.com/canonical/cos-coordinated-workers,PyPI,y,y,"Abstractions for charms following the coordinator-worker pattern, used by Observability Team."
+jujubigdata,legacy,https://pypi.org/project/jujubigdata/,http://pythonhosted.org/jujubigdata/,https://github.com/juju-solutions/jujubigdata,PyPI,,,Legacy library for developing Big Data charms.
+jujuresources,legacy,https://pypi.org/project/jujuresources/,http://pythonhosted.org/jujuresources/,https://github.com/juju-solutions/jujuresources,PyPI,y,y,Legacy library for loading binary resources.
+ops_reactive_interface,,https://pypi.org/project/ops-reactive-interface/,https://github.com/canonical/ops-reactive-interface/tree/main/docs,https://github.com/juju-solutions/ops-reactive-interface,PyPI,y,y,"Helper for relation library developers, to allow an ops based relation library to interact with legacy, reactive charms."
+ops.manifest,,https://pypi.org/project/ops.manifest/,,https://github.com/canonical/ops-lib-manifest,PyPI,,y,Work with Kubernetes manifests.
+sborl,legacy,https://pypi.org/project/sborl/,,https://github.com/canonical/sborl,PyPI,y,y,Legacy library for implementing relation libraries.
+juju_topology,legacy,https://charmhub.io/observability-libs/libraries/juju_topology,,https://github.com/canonical/observability-libs,Charmhub,y,y,Deprecated in favour of ``cosl.juju_topology.JujuTopology``.
+kubernetes_service_patch,legacy,https://charmhub.io/observability-libs/libraries/kubernetes_service_patch,,https://github.com/canonical/observability-libs,Charmhub,,y,Deprecated in favour of ops.Unit.set_ports.
+charm_tracing,legacy,https://charmhub.io/tempo-coordinator-k8s/libraries/charm_tracing,,https://github.com/canonical/tempo-coordinator-k8s-operator,Charmhub,y,y,Consider using ops[tracing] instead.
+data_models,legacy,https://charmhub.io/data-platform-libs/libraries/data_models,,https://github.com/canonical/data-platform-libs,Charmhub,y,y,``pydantic``-based typed relation data – unnecessary with ops 2.23+ due to the addition of typed relation data.
+upgrade,,https://charmhub.io/data-platform-libs/libraries/upgrade,,https://github.com/canonical/data-platform-libs,Charmhub,y,y,Manage in-place upgrades.
+kubernetes_compute_resources_patch,,https://charmhub.io/observability-libs/libraries/kubernetes_compute_resources_patch,,https://github.com/canonical/observability-libs,Charmhub,,y,Patch Kubernetes compute resource limits.
+rollingops,,https://charmhub.io/rolling-ops/libraries/rollingops,,https://github.com/canonical/charm-rolling-ops,Charmhub,y,y,"Perform ""rolling"" operations across units, for example rolling restarts."
```

Comment:
I'm looking into this one in my secondment. It's flawed, and there isn't a replacement version. I'd leave it as-is for now, but likely we'll want to add a `legacy` label to it in the future.

Reply:
Tricky one. "team" doesn't seem right because it's already used by other teams. "experimental" feels like it's in the other direction, going from that to supported.

I would leave it blank for now, and hopefully in a week or so I should have a better answer that can be a follow-up PR.